### ISSUE TITLE
NamedArguments with more than 26 items in arrayListOf

### ISSUE THREAD

**Issue by joancolmenerodev** [detekt/detekt#3438]
The rule or changes you're looking for might have already been suggested!
Please search in the [issues](https://github.com/detekt/detekt/issues) before creating a new one.

## Expected Behavior of the rule
I'm trying to add items on a list 

```
private var myList = arrayListOf<>(
"Example1",
"Example2",
....
"Example26" <-- crashes
)
```
The logs are : 

>complexity - 5min debt
        NamedArguments - [myList] at pathToMyClass.kt:49:34

Since list doesn't have a fixed value, does it make sense to don't let to have more than 26 items as a parameter on `arrayListOf<>()`?

**Comment by schalkms** [comment]
Good point! Thanks for the feedback! I guess it makes sense to ignore this for variable number of arguments (vararg).

**Comment by schalkms** [comment]
@joancolmenerodev do you have time to help us out here and submit a PR?

**Comment by joancolmenerodev** [comment]
> @joancolmenerodev do you have time to help us out here and submit a PR?

Sure, tomorrow I'll try to open a PR

**Comment by joancolmenerodev** [comment]
Hello guys, I've tried to reproduce it on a new branch with tests but the test was not complaining so I've started to see the Spek and I found something similar and I guess this is already fixed on #3294 , I'm not pointing to 1.16.0 milestone we are on 1.15.0.
So I'll close this.

Thanks
